web/pdf_thumbnail_view.js

```diff
@@ -405,11 +423,13 @@ var PDFThumbnailView = (function PDFThumbnailViewClosure() {
     },
   };
 
+  PDFThumbnailView.cleanup = function() {
```

Comment:
Good, I think having that method explicitly defined as static among the other ones would resolve the concern I had here in terms of code readability.